src/platform/zcu102/create_design.tcl

```diff
@@ -0,0 +1,161 @@
+#       NON-INFRINGEMENT, OR FITNESS FOR ANY PARTICULAR PURPOSE; and
+#       (2) Xilinx shall not be liable (whether in contract or tort,
+#       including negligence, or under any other theory of liability) for
+#       any loss or damage of any kind or nature related to, arising under
+#       or in connection with these materials, including for any direct, or
+#       any indirect, special, incidental, or consequential loss or damage
+#       (including loss of data, profits, goodwill, or any type of loss or
+#       damage suffered as a result of any action brought by a third party)
+#       even if such damage or loss was reasonably foreseeable or Xilinx
+#       had been advised of the possibility of the same.
+#
+# Critical Applications:
+#
+#       Xilinx products are not designed or intended to be fail-safe, or
+#       for use in any application requiring fail-safe performance, such as
+#       life-support or safety devices or systems, Class III medical
+#       devices, nuclear facilities, applications related to the deployment
+#       of airbags, or any other applications that could lead to death,
+#       personal injury, or severe property or environmental damage
+#       (individually and collectively, "Critical Applications"). Customer
+#       assumes the sole risk and liability of any use of Xilinx products
+#       in Critical Applications, subject only to applicable laws and
+#       regulations governing limitations on product liability.
+#
+# THIS COPYRIGHT NOTICE AND DISCLAIMER MUST BE RETAINED AS PART OF THIS
+# FILE AT ALL TIMES.
+#
+# *************************************************************************
+
+# Get current directory, used throughout script
+set launchDir [file dirname [file normalize [info script]]]
+set sourcesDir ${launchDir}/sources
+
+# Create the project using board support
+set projName "xilinx_zcu102_dynamic_0_1"
+set projPart "xczu9eg-ffvb1156-2-e"
+set projBoardPart "xilinx.com:zcu102:part0:3.2"
+#set_param board.RepoPaths ${sourcesDir}/boardrepo/samsung/1.0
+create_project $projName ./$projName -part $projPart
+set_property board_part $projBoardPart [current_project]
+
+# Set required environment variables and params
+set_param project.enablePRFlowIPI 1
+set_param project.enablePRFlowIPIOOC 1
+set_param chipscope.enablePRFlow 1
+set_param bd.skipSupportedIPCheck 1
+set_param checkpoint.useBaseFileNamesWhileWritingDCP 1
+
+# Specify and refresh the IP local repo
+set_property ip_repo_paths "${sourcesDir}/iprepo" [current_project]
+update_ip_catalog
+
+# Import HDL, XDC, and other files
+#import_files -norecurse ${sourcesDir}/hdl/CuDmaController.v
+#import_files -norecurse ${sourcesDir}/hdl/CuISR.v
+#import_files -norecurse ${sourcesDir}/hdl/embedded_scheduler_hw.v
+#import_files -norecurse ${sourcesDir}/hdl/iob_static.v
+#import_files -norecurse ${sourcesDir}/hdl/irq_handler.v
+#import_files -norecurse ${sourcesDir}/hdl/isr_irq_handler.sv

```

Comment:
Looks like CuDMAController.v, CuISR.v, embedded_scheduler_hw.v etc are removed in the Tcl script. Why we still check them in?

Reply:
yes will update the sources and this tcl file.